modules/data-mover-for-csi-snapshots.adoc

```diff
@@ -0,0 +1,211 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="data-mover-for-csi-snapshots_{context}"]
+= Data Mover for CSI snapshots
+
+
+The OADP 1.1 Data Mover enables customers to back up CSI volume snapshots to a remote object store and to restore stateful applications from the store in the event of a failure, accidental deletion or corruption of the cluster.
+The OADP 1.1 Data Mover solution uses the Restic option of VolSync.
+[IMPORTANT]
+====
+[subs="attributes+"]
+Data Mover is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[NOTE]
+=====
+Data Mover supports backup and restore of CSI volume snapshots only.
+
+Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
+=====
+
+.Prerequisites
+
+* The `StorageClass` and `VolumeSnapshotClass` CRs support CSI.
+
+* Only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
+
+* Only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
+
+* You have included the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR.
+
+* You have installed the VolSync Operator using the Operator Lifecycle Manager (OLM).
++
+[NOTE]
+====
+The VolSync operator is required only for use with the Technology Preview Data Mover. The operator is not required for using OADP GA features.
+====
+
+* You have installed the OADP operator using OLM.
+
+.Procedure
+
+. Configure a Restic secret:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret-name>
+  namespace: openshift-adp
+type: Opaque
+stringData:
+  # The repository encryption key
+  RESTIC_PASSWORD: <secure-restic-password>
+----
++
+[NOTE]
+====
+By default, the operator looks for a secret named `dm-credential`. If you are using a different name, you need to specify the name via Data Protection Application (DPA) CR using `dpa.spec.features.dataMover.credentialName`.
+====
+. Create a DPA similar to the following example. Note that `defaultPlugins` includes CSI.
```

Comment:
Maybe 'csi` instead of CSI or "Note that the default plug-ins include CSI."

Reply:
Done